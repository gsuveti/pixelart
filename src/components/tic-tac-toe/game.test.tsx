import React from 'react';
import Game from './game';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<Game></Game>)
        .toJSON();

    expect(tree).toMatchSnapshot()
});
