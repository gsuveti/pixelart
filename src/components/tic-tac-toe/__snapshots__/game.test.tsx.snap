// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div
      data-test="name"
    />
    <div
      data-test="role"
    >
      
    </div>
    <div
      data-test="color"
    >
      
    </div>
  </div>
  <div
    className="game"
  >
    <div
      className="game-board"
    >
      <div
        className="sc-htpNat jRiZDM"
      >
        <div
          className="sc-bwzfXH cpnAaf"
        >
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
        </div>
        <div
          className="sc-bwzfXH cpnAaf"
        >
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
        </div>
        <div
          className="sc-bwzfXH cpnAaf"
        >
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
          <button
            className="sc-bdVaJa eAEVDY"
            onClick={[Function]}
          />
        </div>
      </div>
    </div>
    <div
      className="game-info"
    >
      <div>
        Next player: X
      </div>
      <div>
        <div>
          <button
            className="btn"
            onClick={[Function]}
          >
            Go to game start
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
